rdbsync/__init__.py

```diff
@@ -150,17 +152,20 @@ def run(centos_url, fedora_url, token_file, timeout, poll_interval, log_level):
             # Grab a list of duplicate result ids - if some of the results
```

Comment:
The comment still says we are looking up duplicate result ids.
Maybe this should be updated?